### PR TITLE
Add blog about changing logging level of a WildFly application on the Cloud

### PR DESCRIPTION
**Pull request by jmesnil**
Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

### CODE REVIEW

**Comment by jmesnil** [comment]
@jamezp could you please review if the logging commands are correct? thanks!

I planned the post for today but there is no rush. If you don't have time to review it today, I'll change the date to post it later next week?

**Review by yersan — COMMENTED** [review]
One suggestion, maybe you could mention that the approach is only valid if the logging subsystem is present. I mean, in the case of a trimmed server, the logging subsystem could be not installed.

**Review by jamezp — COMMENTED** [review]
This looks good to me @jmesnil on the technical level. @yersan seems to have spotted a few things :)

I don't know if it's worth mentioning, but should there be an example of debugging a specific logger?

```
/subsystem=logging/logger=org.example.web.app:add(level=DEBUG)
```

File: _posts/2021-06-11-change-log-level-wildfly-cloud.adoc [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: How to Change Logging Level for WildFly on the Cloud
+date: 2021-06-11
+tags: wildfly logging cloud openshift kubernetes
+synopsis: This article presents a simple script to change logging level on a WildFly application deployed in Kubernetes or OpenShift.
+author: jmesnil
+---
+
+When your WildFly application is running on the Cloud (on Kubernetes, OpenShift or any other container plaform) and starts to behave unexpectedly, it is useful to be able to change the logging of the server to display debug logs to get a better insight.
```
[the remark this answers is not in the document]
Comment (jmesnil) [reply _posts/2021-06-11-change-log-level-wildfly-cloud.adoc]: fixed, thanks

File: _posts/2021-06-11-change-log-level-wildfly-cloud.adoc [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+08:26:35,004 DEBUG [org.jboss.resteasy.resteasy_jaxrs.i18n] (default task-1) RESTEASY002315: PathInfo: /config/value
+08:26:35,005 DEBUG [org.jboss.resteasy.cdi.i18n] (default task-1) RESTEASY010520: Beans found for class org.wildfly.quickstarts.microprofile.config.ConfigResour
+ce : [Managed Bean [class org.wildfly.quickstarts.microprofile.config.ConfigResource] with qualifiers [@Any @Default]]
+08:26:35,006 DEBUG [org.jboss.resteasy.resteasy_jaxrs.i18n] (default task-1) MessageBodyWriter: org.jboss.resteasy.spi.ResteasyProviderFactory$SortedKey
+08:26:35,006 DEBUG [org.jboss.resteasy.resteasy_jaxrs.i18n] (default task-1) MessageBodyWriter: org.jboss.resteasy.plugins.providers.StringTextStar
+----
+
+## JBOSS_HOME environment variable
+
+These commands works fine whent the application image has been built with https://github.com/wildfly/wildfly-s2i[WildFly S2I] as the `JBOSS_HOME` environment variable is set and we can invoke `$JBOSS_HOME/bin/jboss-cli.sh` without any issue.
```
[the remark this answers is not in the document]
Comment (jmesnil) [reply _posts/2021-06-11-change-log-level-wildfly-cloud.adoc]: fixed, thanks

**Comment by jmesnil** [comment]
@jamezp good suggestion. I added a note that this script is ham fisted and would likely have to be updated to change log level to specific loggers related to the user application.

I've also updated its date to tomorrow (2021-06-15) for publication.

File: _posts/2021-06-11-change-log-level-wildfly-cloud.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (yersan) [reply _posts/2021-06-11-change-log-level-wildfly-cloud.adoc]: Somehow, it has not been changed yet, I still see "whent"

[the remark this answers is not in the document]
Comment (jmesnil) [reply _posts/2021-06-11-change-log-level-wildfly-cloud.adoc]: sorry, I forgot to save :)

File: _posts/2021-06-15-change-log-level-wildfly-cloud.adoc [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+08:26:35,004 DEBUG [org.jboss.resteasy.resteasy_jaxrs.i18n] (default task-1) RESTEASY002315: PathInfo: /config/value
+08:26:35,005 DEBUG [org.jboss.resteasy.cdi.i18n] (default task-1) RESTEASY010520: Beans found for class org.wildfly.quickstarts.microprofile.config.ConfigResour
+ce : [Managed Bean [class org.wildfly.quickstarts.microprofile.config.ConfigResource] with qualifiers [@Any @Default]]
+08:26:35,006 DEBUG [org.jboss.resteasy.resteasy_jaxrs.i18n] (default task-1) MessageBodyWriter: org.jboss.resteasy.spi.ResteasyProviderFactory$SortedKey
+08:26:35,006 DEBUG [org.jboss.resteasy.resteasy_jaxrs.i18n] (default task-1) MessageBodyWriter: org.jboss.resteasy.plugins.providers.StringTextStar
+----
+
+## JBOSS_HOME environment variable
+
+These commands works fine when the application image has been built with https://github.com/wildfly/wildfly-s2i[WildFly S2I] as the `JBOSS_HOME` environment variable is set and we can invoke `$JBOSS_HOME/bin/jboss-cli.sh` without any issue.
```
Comment (yersan) [inline _posts/2021-06-15-change-log-level-wildfly-cloud.adoc:56]: s/These commands works/These commands work

**Review by yersan — APPROVED** [review]
Ups, just saw an additional minor more, with that one fixed, the content looks fine to me.

File: _posts/2021-06-15-change-log-level-wildfly-cloud.adoc (hunk 3 above)
Comment (jmesnil) [reply _posts/2021-06-15-change-log-level-wildfly-cloud.adoc]: thanks!

**Review by jamezp — APPROVED** [review]

**Pull request merged**